test/pylib/coverage_utils.py

```diff
@@ -0,0 +1,1293 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2015-present ScyllaDB
```

Comment:
I'm sure it didn't take 9 years to write.

Reply:
will fix